qa-tests-backend/src/test/groovy/AuthServiceTest.groovy

```diff
@@ -26,13 +26,13 @@ class AuthServiceTest extends BaseSpecification {
         assert status
         assert status.userId == "admin"
 
-        status.authProvider.withDo {
+        status.authProvider.with {
```

Comment:
Probably instead of `with` we should use `verifyAll` https://spockframework.org/spock/docs/2.3/spock_primer.html#_using_verifyall_to_assert_multiple_expectations_together

https://github.com/stackrox/stackrox/blob/df98e402d4f8d87175822493de923973c2f3a51d/qa-tests-backend/src/test/groovy/DeclarativeConfigTest.groovy#L312-L318